src/Stringy.php

```diff
@@ -427,7 +427,7 @@ public function toAscii()
     {
         $str = $this->str;
         $charsArray = array(
-            'a'    => array('à', 'á', 'â', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
+            'a'    => array('à', 'á', 'â', 'ä', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
```

Comment:
@vlakoff No worries! I would have done the same. Cleaning all that up is for a separate commit. https://github.com/danielstjules/Stringy/commit/91d8da474190185d1f1cdca9b183359209da32b4 I only fixed alignment - haven't touched order.